Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -18,49 +18,57 @@
             name="google-site-verification"
             content="doAsS2f0kVQcQMjZcJuo2Pdj60GkhXI2iKbwfaUBImk"
         />
-
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" integrity="sha512-3n19xznO0ubPpSwYCRRBgHh63DrV+bdZfHK52b1esvId4GsfwStQNPJFjeQos2h3JwCmZl0/LgLxSKMAI55hgw==" crossorigin="anonymous">
+        </script>
         <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
         <script type="text/javascript">
-            (function (p, l, o, w, i, n, g) {
-                if (!p[i]) {
-                    p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
-                    p.GlobalSnowplowNamespace.push(i);
-                    p[i] = function () {
-                        (p[i].q = p[i].q || []).push(arguments);
-                    };
-                    p[i].q = p[i].q || [];
-                    n = l.createElement(o);
-                    g = l.getElementsByTagName(o)[0];
-                    n.async = 1;
-                    n.src = w;
-                    g.parentNode.insertBefore(n, g);
+            function detectBrowser() {
```

Comment:
This conditionally switches the snowplow script, no need for that. Lets move the detectBrowser to its own script.

Reply:
Looking bvetter! Did you check it works if it deploys? i.e. npm run build

Reply:
yes, npm run build runs perfectly. It generates the JS and copies it to the dist folder.

Reply:
right, and the index file has access to that path?

Reply:
> right, and the index file has access to that path?
yes, the modernizr-custom.js is downloaded in both chrome and ie per my tests in local. Do you know any ways that we can do to make sure if it is accessible from dist ?